dynatrace-operator-google-marketplace/chart/dynatrace-operator/templates/configmap.yaml

```diff
@@ -2855,7 +2855,7 @@ data:
 
       {{- if .Values.proxy }}
       proxy:
-        valueFrom: {{ .Values.name }}
```

Comment:
the value for proxy is loaded from a config map that has the same `name` as the dynakube